README.md

```diff
@@ -27,25 +27,13 @@ Originally forked from [megabytesofrem/pridefetch](https://github.com/megabyteso
 **Requirements:** `Python 3.7` or higher and `python-distro`
 ```bash
 git clone https://github.com/SpyHoodle/pridefetch.git
-cd pridefetch
-chmod +x pridefetch
-```
-
-Then, run pridefetch
-
-```bash
-./pridefetch
-```
-
-You can also add pridefetch to your `$PATH` to run it anywhere<br>
```

Comment:
Can we give an alternative to this (i.e. telling people how to zip it?)

Reply:
It would be complicated - they may already be in the pridefetch/src directory, or may be in the pridefetch directory, or may have gotten out of it - I don't want to duplicate the first two steps either, and having multiple messages about requirements such as requiring zip to be installed, and which directory to be in would be annoying, and likely missed by someone reading it quickly.

Zipping too would be complicated, you have to be in the pridefetch directory, then run a command, then run some extra command to add a shebang and make it a unix executable, then move it to /usr/bin.

May I suggest a mention to the releases tab, or you could show me how you'd implement the installation process.

However, I would suggest a future solution being an INSTALL.md file/similar, which lists different repositories, and also how to install from source. It'd separate everything out, making code look less duplicated.

Reply:
Alright, that's fine
Perhaps in a future commit we could make a "package.sh" script/similar? That would alleviate the complexity while also allowing us to cd relative to the script directory

I'm going to approve this, but I'll also create an issue for it

Reply:
Some repositories do typically have a `install.sh` instead of just a `package.sh`. I suggest we work & talk on this in the issue.